modules/node-observability-scripting.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-observability-scripting_{context}"]
+= Node Observability Operator Scripting
+
+[About Scripting]
+
+.Prerequisites
+
+* You have installed the Node Observability Operator.
+* You have created the `NodeObservability` custom resource (CR).
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `NodeObservabilityScripting` resource file named `nodeobservabilityrun.yaml` that contains the following text:
++
+[source,yaml]
+----
+apiVersion: nodeobservability.olm.openshift.io/v1alpha2
+kind: NodeObservabilityScripting
+metadata:
+  name: nodeobservabilityscripting
+spec:
+  nodeObservabilityRef:
+    name: cluster
+----
+
+. Build a container using the `Docker.dev` file.
```

Comment:
The whole section after [IMPORTANT] can be replaced as in 2,3,4 of the CRIO Profiling Procedure section

Reply:
Understood.